intersection-observer/META.yml

```diff
@@ -11,3 +11,7 @@ links:
           subtest: IntersectionObserverEntry interface object length
         - test: idlharness.window.html
           subtest: 'IntersectionObserver interface: observer must inherit property "thresholds" with the proper type'
+    - product: chrome
+      url: https://crbug.com/870947
```

Comment:
Confirmed this one. This bug could be turned into something more descriptive since there's only a single subtest failure, but it's the right bug.